src/accelerate/scheduler.py

```diff
@@ -42,10 +42,24 @@ class AcceleratedScheduler:
             Whether or not the dataloaders split one batch across the different processes (so batch size is the same
             regardless of the number of processes) or create batches on each process (so batch size is the original
             batch size multiplied by the number of processes).
+        adjust_scheduler_to_accumulation (`bool`, *optional*, defaults to `False`):
```

Comment:
```suggestion
        adjust_to_accumulation (`bool`, *optional*, defaults to `False`):
```
We are in a scheduler.